superset/viz.py

```diff
@@ -2113,17 +2121,13 @@ def process_spatial_data_obj(self, key, df):
         elif spatial.get('type') == 'delimited':
             lon_lat_col = spatial.get('lonlatCol')
             df[key] = df[lon_lat_col].apply(self.parse_coordinates)
-
             if spatial.get('reverseCheckbox'):
-                df[key] = [
-                    tuple(reversed(o)) if isinstance(o, (list, tuple)) else (0, 0)
```

Comment:
I'm happy we're getting rid of the `(0, 0)` here.

Reply:
yeah that place on the west coast of africa that throws the autozoom off...